app/vlselect/main.go

```diff
@@ -73,10 +76,30 @@ var vmuiFileServer = http.FileServer(http.FS(vmuiFiles))
 func RequestHandler(w http.ResponseWriter, r *http.Request) bool {
 	path := r.URL.Path
 
-	if !strings.HasPrefix(path, "/select/") && !strings.HasPrefix(path, "/internal/select/") {
-		// Skip requests, which do not start with /select/, since these aren't our requests.
+	switch {
+	case strings.HasPrefix(path, "/internal/select/") || path == "/internal/select":
+		if *disableInternal {
+			httpserver.Errorf(w, r, "requests to /internal/select/* are disabled with -internalselect.disable command-line flag")
+			return true
+		}
+		fallthrough
```

Comment:
It is better from readability and maintainability PoV calling the `internalselect.RequestHandler()` here instead of moving it to the bottom of this function